extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -41,6 +42,7 @@ AMP.registerServiceForDoc(
 installActivityService(AMP.win);
 installCidService(AMP.win);
 installCryptoService(AMP.win);
+variableServiceFor(AMP.win);
```

Comment:
Ideally we wouldn't expose this as a service, but instead just create it as an instance var of `Instrumentation` class.